docker-compose.yml

```diff
@@ -9,6 +9,9 @@
                                               # will cause persistence issues!!
       "./observatory-extension:/observatory-extension"
     ],
+    "ports": [
+      "5432:5432"
+    ],
```

Comment:
OK -- that's fine, but I'd rather keep it closed down on our test deployment.  Could you remove this and keep the change locally only?